local_server.sh

```diff
@@ -25,6 +25,9 @@ redis-server config/redis_local.conf &
 echo "Starting compass watch"
 compass watch server &
 
+echo "Starting jsx compiler"
+jsx --watch -x jsx server/static/jsx/ server/static/js/ &
```

Comment:
Don't install this globally - global dependencies are always a huge PITA to manage. Just do a regular `npm install`, then use `node_modules/.bin/jsx` Instead.

I also think this is missing a compilation step for production? Where does that happen?

It's also really important that we use the dev version of React in dev, and the prod version of React in prod. The prod version is **way** faster, and the dev version gives **way** more helpful error messages.